lib/widgets/item_cocktail.dart

```diff
@@ -8,40 +10,68 @@ class CocktailItem extends StatelessWidget {
   const CocktailItem({
     Key? key,
     required this.name,
+    this.isTypeOfGlass = false,
   }) : super(key: key);
 
   final String name;
+  final bool? isTypeOfGlass;
+
+  String getImage() {
+    if (isTypeOfGlass!) {
+      return Assets.glass;
+    } else {
+      return Assets.categories;
+    }
+  }
```

Comment:
We can simplify this function like:
```suggestion
  String get image =>  isTypeOfGlass ? Assets.glass : Assets.categories;
```